shared-bindings/_stage/__init__.c

```diff
@@ -89,6 +89,10 @@ STATIC mp_obj_t stage_render(size_t n_args, const mp_obj_t *args) {
         mp_raise_TypeError(translate("argument num/types mismatch"));
     }
     displayio_display_obj_t *display = MP_OBJ_TO_PTR(native_display);
+    uint8_t scale = 1;
+    if (n_args >= 8) {
```

Comment:
`scale` is missing from the arg documentation above.

Reply:
Oops, thanks.